src/models/SessionEvent.ts

```diff
@@ -71,4 +70,63 @@ export class SessionEvent {
 
 	@IsBoolean()
 	notified!: boolean;
+
+}
+
+/**
+ * Object to represent data contained in SessionEvent messages sent by this lambda
+ */
+export class ExtSessionEvent extends SessionEvent {
```

Comment:
Would you please be able to name this / update the comment to reflect what the difference is between this and `SessionEvent` so that it's obvious to the reader?